server/routes/viewingLicences/handlers/viewLicence.ts

```diff
@@ -61,8 +61,10 @@ export default class ViewAndPrintLicenceRoutes {
         )
       }
 
+      const additionalLicenceConditions = licence.additionalLicenceConditions as AdditionalCondition[]
```

Comment:
Casting the type here to fix the typing issue I was having on ln 67 below. Was erroring with `Argument of type 'string' is not assignable to parameter of type 'never'.`

Reply:
Fix no longer needed